Adyen/Assets/zh-CN.lproj/Localizable.strings

```diff
@@ -31,7 +32,7 @@
 "adyen.card.stored.expires" = "%@过期";
 "adyen.dropIn.stored.title" = "确认 %@ 支付";
 "adyen.dropIn.preselected.openAll.title" = "更改支付方式";
-"adyen.continueTo" = "继续至%@";
+"adyen.continueTo" = "继续至 %@";
```

Comment:
This is correct. We have this space in translation and other strings have spaces